src/types/auth.d.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-self-import */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable camelcase */
+
+import {ObjectID} from 'mongodb';
+
+export interface DecodedIdToken {
+  aud: string;
+  auth_time: number;
+  email?: string;
+  email_verified?: boolean;
```

Comment:
@riteshsp2000 how will we be getting this email_verified boolean?

Reply:
This is just a type definition for the decodedToken object that we get from firebase after verifying the JWT. For some reason the firebase admin types aren't exported which is why I had to create this one.